tests/e2e/autoscaling/00-create-vm.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- ../../../deploy/autoscale-scheduler.yaml
+- ../../../deploy/autoscaler-agent.yaml
+unitTest: false
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: example
```

Comment:
Why do we need a service here?

Reply:
I've copied this example from `neonvm`. 
Without service it doesn't work: `example:5432 - no response`, I've removed extra stuff that we don't really need (like a bunch of ports and vector), we could add it back if needed (or for other tests)

Reply:
Ahhh I see - didn't realize it was used for providing a stable address. makes sense